tls/s2n_tls13_handshake.c

```diff
@@ -105,25 +105,31 @@ int s2n_tls13_compute_pq_hybrid_shared_secret(struct s2n_connection *conn, struc
     notnull_check(client_ecc_params);
 
     DEFER_CLEANUP(struct s2n_blob ecdhe_shared_secret = { 0 }, s2n_blob_zeroize_free);
+    struct s2n_blob *pq_shared_secret;
 
+    struct s2n_kem_params *server_kem_params = &server_kem_group_params->kem_params;
+    notnull_check(server_kem_params);
```

Comment:
I think these two lines can be moved inside the `else` statement

Reply:
Actually, I don't even need those lines, I can directly assign the shared secret in the `else` clause (all the necessary `notnull_check`s were done above).